gym_ai2thor/envs/ai2thor_env.py

```diff
@@ -74,6 +74,11 @@ def __init__(self, seed=None, config_file='config_files/config_example.json', co
                                             shape=(channels, self.config['resolution'][0],
                                                    self.config['resolution'][1]),
                                             dtype=np.uint8)
+        # Rendering options
+        self.render_options = defaultdict(lambda: False)
+        if 'render_options' in self.config:
+            for option, value in self.config['render_options'].items():
+                self.render_options[option] = value
```

Comment:
Wow, I was wrong, the first time you access anything e.g.
`self.render_options['fsdaf;eangeaneagea'] == False`
But maybe this isn't too clear? maybe  I'm happy to leave it

Reply:
I set the defaultdict, builtin from python, so that if there is a KeyError you get 'False' automatically no matter which key. Standard and expected python behaviour, but let me know if you want me to document something there.

Reply:
add one comment `# set segmentation and bounding box options off as default`

Reply:
Added comment